pipeline/data_store.py

```diff
@@ -24,6 +24,7 @@
     'detection': ['subtraction'],
     'cutting': ['detection'],
     'measurement': ['detection', 'photo_cal'],
+    'warp': [ 'photo_cal' ]
```

Comment:
I know dictionaries are not ordered but can we move this process between photo_cal and subtraction?

Reply:
Yeah, will do.

Turns out dictionaries *are* ordered.  I was surprised to find that out, since I'd never treated them that way.